templates/k8smetricsservicemonitor.go

```diff
@@ -35,7 +34,7 @@ var params = map[string][]string{
 	},
 }
 
-var k8sMetricsServiceMonitorAuthSecret = "k8s-metrics-service-monitor-auth"
+var bearerTokenFile = "/var/run/secrets/kubernetes.io/serviceaccount/token"
```

Comment:
First, this is a const, not a var.
Second, this value is used only once in the template, and nowhere else. 
Let's embed it in the template!

Reply:
agreed! done.